frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -13,16 +17,50 @@ export type RevisionsOverviewListProps = {
   service: K8sResourceKind;
 };
 
+const MAX_REVISIONS: number = 3;
+
 const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions, service }) => {
   const canSetTrafficDistribution = useAccessReview({
     group: ServiceModel.apiGroup,
     resource: ServiceModel.plural,
     namespace: service.metadata.namespace,
     verb: 'update',
   });
+
+  const namespace = service.metadata?.namespace;
+  const traffic = service.status?.traffic;
+  const name = service.metadata?.name;
+
+  const filteredRevisions = (): K8sResourceKind[] => {
+    if (!revisions || !revisions.length) {
+      return [];
+    }
+    const [revWithTraffic, revWithoutTraffic] = _.partition(revisions, (element) => {
+      return traffic ? _.find(traffic, { revisionName: element.metadata?.name }) : false;
+    }).map((element) => _.sortBy(element, ['metadata.creationTimestamp']));
+    return revWithTraffic.length < MAX_REVISIONS
+      ? _.concat(revWithTraffic, revWithoutTraffic.slice(0, MAX_REVISIONS - revWithTraffic.length))
+      : revWithTraffic;
+  };
+
+  const getRevisionsLink = () => {
+    const url = `/search/ns/${namespace}`;
+    const searchQuery = `serving.knative.dev/service=${name}`;
+    const params = new URLSearchParams();
+    params.append('kind', referenceForModel(RevisionModel));
+    params.append('q', searchQuery);
+    history.push(`${url}?${params.toString()}`);
+  };
+
   return (
     <>
       <SidebarSectionHeading text="Revisions" className="revision-overview-list">
+        {revisions?.length > MAX_REVISIONS && (
+          <Button type="button" variant="link" onClick={getRevisionsLink}>
```

Comment:
@nemesis09 this should be `link` as we have for pods.

<img width="798" alt="Screenshot 2020-02-17 at 7 54 02 PM" src="https://user-images.githubusercontent.com/5129024/74662262-9ba35f00-51bf-11ea-97d9-09849520c865.png">

Reply:
updated code to use link